### DISCUSSION TITLE
platform选项中没有wasm

### DISCUSSION THREAD

**Discussion by Hyiker** [xmake-io/xmake-vscode#234] [Q&A]
![image](https://github.com/xmake-io/xmake-vscode/assets/28663016/9714c81e-e1b2-4741-ab82-eae9e8d45a11)
目前版本中似乎没有wasm等platform的选择，导致如果配置`xmake.additionalConfigArguments`加入`-p wasm`了运行configure命令生成的command如下：

```bash
 xmake f -p windows -a x64 -m debug -p wasm -c
```

**Comment by waruqi** [comment]
可以直接提个 pr 过来

**Reply by Hyiker** [reply]
实现过程中发现现在xmake似乎没有提供可以直接获取platform/arch的命令，只有在`xmake f --help`中会列举对应的支持列表，但是目前语义较为混乱，有考虑提供统一的接口枚举么

**Reply by waruqi** [reply]
原本就是枚举的 https://github.com/xmake-io/xmake-vscode/blob/6574cff2323cf3eee3d922d11ae77aa59c718535/src/xmake.ts#L1248

**Reply by waruqi** [reply]
其他的 可以参考这个枚举
